aiohttp/http_writer.py

```diff
@@ -71,10 +70,10 @@ def _write(self, chunk: bytes) -> None:
         size = len(chunk)
         self.buffer_size += size
         self.output_size += size
-
-        if self._transport is None or self._transport.is_closing():
+        transport = self._protocol.transport
```

Comment:
Sounds like now it's just
```suggestion
        transport = self.transport
```